x-pack/plugins/canvas/canvas_plugin_src/functions/common/all.ts

```diff
@@ -4,7 +4,13 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export function all() {
+import { FunctionSpec } from '../types';
+
+export interface Arguments {
```

Comment:
Maybe it's merely a stylistic thing: I see a bunch of (non-exported) `Arguments` interfaces, all specific to the function at hand eg. for `Markdown`, `URLParam` etc. Here however it's exported, I don't immediately see if it's used anywhere

Reply:
I think it's a remainder from a previous diff... I'll remove the exports.